app/templates/introduction.html

```diff
@@ -40,30 +50,39 @@ <h2 class="saturn">Your response is legally required</h2>
         </div>
       </div>
     </div>
-    <div>
-      {% if content.information_to_provide %}
-      <h2 class="saturn">You will be asked to provide information for the business, including:</h2>
-      <ul>
-        {% for information_item in content.information_to_provide %}
-          <li>{{ information_item }}</li>
-        {% endfor %}
-      </ul>
-      {% endif %}
-      <div class="notice u-mb-s u-mb-m@s">
-        <div class="notice__text venus">If actual figures are not available, please provide informed estimates.</div>
-      </div>
 
-    </div>
+  {% endblock business_details %}
 
-    <form class="form" method="POST" novalidate>
-      <button class="btn btn--primary btn--lg qa-btn-get-started" type="submit" name="action[start_questionnaire]">Start survey</button>
-    </form>
-    <div class="lock u-mb-s u-mb-m@s">
-      <div class="venus lock__text">We will treat your data securely and confidentially</div>
-    </div>
-    <h2 class="saturn">Legal Information</h2>
-    <p class="mars">You are required to complete this questionnaire. If you do not complete and return this questionnaire by {{ meta.survey.return_by|format_date }}, penalties may be incurred (under section 4 of the Statistics of Trade Act 1947).</p>
+  {% block information_to_provide %}
+
+  <div class="u-mb-l">
+    {% if content.information_to_provide %}
+    <h2 class="saturn">You will be asked to provide information for the business, including:</h2>
```

Comment:
From the Google slides it looks like `the business` should be the piped business name for UKIS. If that is the case, maybe we could do that for all surveys?